tools/launchdev.sh

```diff
@@ -134,6 +125,12 @@ function st2start(){
         fi
     done
 
+    # Register sensors, actions and rules
+    echo 'Registering sensors, actions and rules...'
```

Comment:
Yeah, good change. I believe we did this before, but at some point we changed it back (don't remember why anymore).